rdbms/src/test/java/org/spine3/server/storage/jdbc/JdbcStorageFactoryShould.java

```diff
@@ -61,6 +62,20 @@ public void setUpTest() {
     }
 
     @Test
+    public void allow_to_use_custom_data_source(){
+        final String dbUrl = newInMemoryDbUrl("factoryTests");
+        final DataSourceConfig config = DataSourceConfig.newBuilder()
+                .setJdbcUrl(dbUrl)
+                .setUsername("SA")
+                .setPassword("pwd")
+                .setMaxPoolSize(12)
+                .build();
+        final HikariConfig hikariConfig = DefaultDataSourceConfigConverter.convert(config);
+        final JdbcStorageFactory factory = JdbcStorageFactory.newInstance(new HikariDataSource(hikariConfig), false);
```

Comment:
I think we can just pass `mock(DataSource.class)`, can`t we? Because Hikari is the default implementation.